front/app/components/UI/ImagesDropzone/index.tsx

```diff
@@ -180,7 +179,7 @@ const RemoveButton = styled.button`
   position: absolute;
   top: 8px;
   right: 8px;
-  z-index: 0;
+  z-index: 1;
```

Comment:
Have delete button of the uploaded image on top of the newly added overlay (with the landing page header image).